Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -430,7 +430,11 @@ - (NSDictionary*) v2dictionary {
     addString(osVersion,            os_version);
     addString(extensionType,        environment);
     addString(vendorId,             idfv);
-    addString(adId,                 idfa);
+    if ([BNCPreferenceHelper preferenceHelper].isDebug) {
```

Comment:
Not sure if there any other condition we cannot get `adId`? if so can we add `unidentified_device` for that case also

Reply:
If we can't get idfa in other cases it's because of privacy settings.  In that case we use idfv instead, and that's legit. So it isn't an `unidentified_device` device in that case, it's just a less identified device.

Reply:
👍